evm_arithmetization/src/generation/mod.rs

```diff
@@ -37,6 +38,9 @@ mod trie_extractor;
 use self::state::State;
 use crate::witness::util::mem_write_log;
 
+#[cfg(test)]
+mod test_generation;
```

Comment:
The test module name is a bit confusing, we don't test anything related to the `GenerationInputs` but just testing hardcoded strings hashing.

Reply:
You can just embed test module at the end of `mod.rs` same as on other places
```
#[cfg(test)]
mod tests {
...
}
```

Reply:
Naming is hard and the module, I am testing, is called `generation`.  The reason I separated the tests out in the last commit was to avoid the super long lines (1000+ cols) from the test input.

Reply:
> Naming is hard and the module, I am testing, is called generation.

Yes, but the feature being tested is unrelated to the `GenerationInputs`. FWIW, it could be very well defined in the `mpt` module where the different transaction types are defined and tested there. The issue is long term codebase consistency, which makes it harder to navigate and reason about for devs.


> The reason I separated the tests out in the last commit was to avoid the super long lines (1000+ cols) from the test input.

I'm sorry but I didn't understand the rationale. These are not wrapped over hundreds of lines so this won't be a problem on most IDEs? Especially as we still have the gigantic lines in the test file anyway.